src/main/java/org/testng/IAnnotationTransformer2.java

```diff
@@ -12,35 +12,42 @@
  * annotation besides @Test.
  */
 public interface IAnnotationTransformer2 extends IAnnotationTransformer {
```

Comment:
Now that we are talking about default methods, we should have moved the methods within this interface to its parent viz. `IAnnotationTransformer` and deprecated off this interface. There's now really no use for having `IAnnotationTransformer2`